tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -229,9 +239,121 @@ setup() {
     assert_rootfs_count "$node" "$sandbox_id" "1"
 }
 
+function setup_kbs_credentials() {
+    image=$1
+    user=$2
+    password=$3
+
+    if [ "${KBS}" = "false" ]; then
+        skip "Test skipped as KBS not setup"
+    fi
+
+    if [[ -z ${user} || -z ${password} ]]; then
+        if [[ -n ${GITHUB_ACTION} ]]; then
+            die "User and/or password not supplied to authenticated registry test"
+        else
+            skip "running local and missing user/password"
+        fi
+    fi
+
+    REGISTRY_CREDENTIAL_ENCODED=$(echo "${user}:${password}" | base64 -w 0)
+
+    auth_json=$(echo "{
+        "auths": {
+            "${image}": {
```

Comment:
Hi @stevenhorsman !

According to https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md#format we will need to strip out any tag in the ${image} reference.

Reply:
Thanks, I've used your cut suggestion from earlier here as well, so hopefully that will work